publisher/product/ontology/build-about.sh

```diff
@@ -65,7 +57,7 @@ __HERE__
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> 
 @prefix owl: <http://www.w3.org/2002/07/owl#> 
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#>
-<${tag_root_url}/LoadFIBODev> a owl:Ontology;
+<${tag_root_url}/Load${ONTPUB_FAMILY}Dev> a owl:Ontology;
```

Comment:
Should be uppercase:
`<${tag_root_url}/Load${ONTPUB_FAMILY^^}Dev> a owl:Ontology;`